.travis.yml

```diff
@@ -34,15 +34,29 @@ matrix:
       after_failure:
         - docker-compose logs
     - language: generic
-      name: "StableX e2e Tests"
+      name: "StableX e2e Tests (Ganache)"
       cache: 
         directories:
           - dex-contracts/node_modules/
       install:
         - ./test/setup_contracts.sh
       script:
         - docker-compose up -d stablex
-        - ./test/e2e-tests-stablex.sh
+        - ./test/e2e-tests-stablex-ganache.sh
+      after_failure:
+        - docker-compose logs
+    - language: generic
+      name: "StableX e2e Tests (Rinkeby)"
+      cache: 
+        directories:
+          - dex-contracts/node_modules/
+      install:
+        - cd dex-contracts && npm ci && npx truffle compile && npm run networks-inject && cd ..
+        - echo -e $GITLAB_PRIVATE_KEY > .ssh/id_rsa && chmod 0500 .ssh/id_rsa
```

Comment:
This doesn't show up in logs anywhere right?

Reply:
No, just the command as far as I can tell: 

https://travis-ci.org/gnosis/dex-services/jobs/617208147#L535